cfgov/jinja2/v1/landing-page-2/index.html

```diff
@@ -79,10 +92,36 @@
 
         </div>
     </div>
+
+    {% if useSideNavLayout == true %}
+    <div class='block
+                    block__bg
+                    block__flush-sides
+                    block__flush-bottom
+                    block__border-top
+                    block__border-right
+                    related_content
+                    '>
+        {# TODO: Add sidebar organisms. #}
+        {{ related_posts.render({
+            'post': get_document('posts', 'cfpbs-new-rule-will-shine-light-on-mortgage-market-practices')
+        }) }}
+    </div>
+    {% endif %}
 {% endblock %}
 
+
+{% block content_sidebar_modifiers -%}
+    o-content-sidebar
```

Comment:
I'll remove as it's not needed in this PR but it's being used in https://github.com/cfpb/cfgov-refresh/blob/email_signup_form/cfgov/preprocessed/css/organisms/content-sidebar.less